buildSrc/src/main/groovy/de/mobilej/unmock/UnMockPlugin.groovy

```diff
@@ -74,7 +74,7 @@ class UnMockPlugin implements Plugin<Project> {
         project.afterEvaluate {
             project.tasks.each {
                 task ->
-                    if (task.name ==~ /.*[cC]ompile.*UnitTest.*/) {
```

Comment:
Hi, @bjoernQ , can you please describe, for which tasks this pattern was incorrect? 
The new pattern `/.*[cC]ompile.*/` is not precise enough. It adds unnecessary dependencies to all "compile" tasks which are not related to tests. And this influence a build time in multi-module project.

Reply:
Thanks for pointing this out. As far as I can remember there are tasks like "compileDebugUnitTestKotlin" and others which need to be covered. Sure the current is a bit broad and I will try to narrow that.

Reply:
Sorry "compileDebugUnitTestKotlin" is certainly matched but there are other tasks added by kapt which are not matched but should. I will look them up and post it here.